docusaurus.config.js

```diff
@@ -88,6 +88,16 @@ const config = {
           target: "_self",
         },
         items: [
+          {
+            href: "/services/sdk/overview?sdk=ts",
+            label: "Typescript SDK",
+            position: "right",
+          },
+          {
+            to: "/services/sdk/overview?sdk=java",
+            label: "Java SDK",
```

Comment:
Ditto "Java / Kotlin SDK"?

Reply:
Added an issue for writing Kotlin docs in general: https://github.com/restatedev/documentation/issues/227
We can take this approach where it's similar for both. But for example, the example on the overview page will look different.